# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gimp/freetype.patch
# Copyright (C) 2014 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- gimp-2.8.10/app/text/gimpfont.c.vanilla	2014-06-14 02:57:36.200770831 +0000
+++ gimp-2.8.10/app/text/gimpfont.c	2014-06-14 02:57:42.192770519 +0000
@@ -27,7 +27,7 @@
 
 #define PANGO_ENABLE_ENGINE  1   /* Argh */
 #include <pango/pango-ot.h>
-#include <freetype/tttables.h>
+#include <tttables.h>
 
 #include "text-types.h"
 
